src/Octoshift/Services/OctoLogger.cs

```diff
@@ -11,7 +11,6 @@ internal static class LogLevel
     public const string INFO = "INFO";
     public const string WARNING = "WARNING";
     public const string ERROR = "ERROR";
-    public const string SUCCESS = "INFO";
```

Comment:
There is no need to delete this.

Reply:
@ArinGhazarian Thank you for reviewing this ❤️ 

It looks like that `SUCCESS` is [non-standard log level in C#](https://learn.microsoft.com/en-us/dotnet/api/microsoft.extensions.logging.loglevel?view=dotnet-plat-ext-7.0#fields), so I tried to standardize the log level.

The `SUCCESS` level is used only  in `LogSuccess` method, I feel like `INFO` is enough because of the method name. 

Could you please let me know why that non-standard log level was made here? 🙇 

Thanks!

Reply:
You're probably right that it's unnecessary. But we'd prefer to keep PR's focused on changing one thing per PR. A PR to change the timestamp format, and a separate PR to improve the log levels would be preferred.

Reply:
My main point was also not to mix arbitrary/unrelated refactors in PRs. But to answer your question, `OctoLogger` is a custom logger that we created for the cli, and those log levels are not following .NET's log levels. Actually success means something to us and we display it with a green text to indicate a success. IMO the success log level is needed, the only thing is we should probably change the constant value from `INFO` to `SUCCESS` but it shouldn't be done in this PR.

Reply:
@dylan-smith @ArinGhazarian thank you both for your kind response ❤️ 

> Actually success means something to us and we display it with a green text to indicate a success.

It does make sense 👍 Thank you so much for letting me know it 💖 

I'm going to remove the changes for `SUCCESS` level 👍

Reply:
Feel free to follow up with another PR to change the constant! ❤️ We really appreciate your contributions.